examples/bert/run_pretraining.py

```diff
@@ -291,6 +301,53 @@ def train_step(self, data):
         self.nsp_accuracy.update_state(data["next_sentence_labels"], nsp_preds)
         return {m.name: m.result() for m in self.metrics}
 
+class AdamLRSchedule(keras.optimizers.schedules.LearningRateSchedule):
+    """
+    Implements a Learning Rate Scheduler for Adam with Linear Warmup and
```

Comment:
In general dostring should have a short oneliner:

Implements a learning rate schedule for BERT.

Optionally followed by more text after an empty line.

Reply:
Alright, I'll change it